plugins/main/public/components/common/welcome/components/vuls_panel/vuls_welcome_panel.tsx

```diff
@@ -0,0 +1,204 @@
+import React, { Fragment, useEffect, useState } from 'react';
+import { VulsTopPackageTable } from '../top_packages_table';
+import VulsSeverityStat from '../vuls_severity_stat/vuls_severity_stat';
+import {
+  PatternDataSourceFilterManager,
+  PatternDataSource,
+  tParsedIndexPattern,
+  useDataSource,
+  FILTER_OPERATOR,
+  VulnerabilitiesDataSourceRepository,
+  VulnerabilitiesDataSource,
+} from '../../../data-source';
+import { severities } from '../../../../../controllers/overview/components/last-alerts-stat/last-alerts-stat';
+import { getCore } from '../../../../../kibana-services';
+import { RedirectAppLinks } from '../../../../../../../../src/plugins/opensearch_dashboards_react/public';
+import { vulnerabilityDetection } from '../../../../../utils/applications';
+import NavigationService from '../../../../../react-services/navigation-service';
+import { WzLink } from '../../../../../components/wz-link/wz-link';
+import { withErrorBoundary } from '../../../../common/hocs';
+import { compose } from 'redux';
+import { withVulnerabilitiesStateDataSource } from '../../../../../components/overview/vulnerabilities/common/hocs/validate-vulnerabilities-states-index-pattern';
+
+const VulsPanelContent = ({ agent }) => {
+  const {
+    dataSource,
+    isLoading: isDataSourceLoading,
+    fetchData,
+  } = useDataSource<tParsedIndexPattern, PatternDataSource>({
+    DataSource: VulnerabilitiesDataSource,
+    repository: new VulnerabilitiesDataSourceRepository(),
+  });
+
+  const [isLoading, setIsLoading] = useState(true);
+  const [severityStats, setSeverityStats] = useState(null);
+  const [topPackagesData, setTopPackagesData] = useState([]);
+
+  const fetchSeverityStatsData = async () => {
+    const data = await fetchData({
+      aggs: {
+        severity: {
+          terms: {
+            field: 'vulnerability.severity',
+            size: 5,
+            order: {
+              _count: 'desc',
+            },
+          },
+        },
+      },
+    });
+    setSeverityStats(data.aggregations.severity.buckets);
+  };
+
+  const fetchTopPackagesData = async () => {
+    fetchData({
+      aggs: {
+        package: {
+          terms: {
+            field: 'package.name',
+            size: 5,
+            order: {
+              _count: 'desc',
+            },
+          },
+        },
+      },
+    }).then(results => {
+      setTopPackagesData(results.aggregations.package.buckets);
+    });
+  };
+
+  useEffect(() => {
+    if (isDataSourceLoading) {
+      return;
+    }
+    setIsLoading(true);
+    Promise.all([fetchSeverityStatsData(), fetchTopPackagesData()])
+      .catch(error => {
+        // ToDo: Handle error
+        console.error(error);
+      })
+      .finally(() => {
+        setIsLoading(false);
+      });
+  }, [isDataSourceLoading, agent.id]);
+
+  const getSeverityValue = severity => {
+    const value =
+      severityStats?.find(v => v.key.toUpperCase() === severity.toUpperCase())
+        ?.doc_count || '0';
```

Comment:
This is because the query to obtain the kpis if it does not return results, the 4 selected values ​​must still be displayed. In this case the severity, which would be Low, Medium, High and Critical. If the request is made correctly and there is no error, it means that it has 0. We need to show the 4 severities.